src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Validators/ClarificationPageValidator.cs

```diff
@@ -11,13 +13,16 @@ public class ClarificationPageValidator : IClarificationPageValidator
     {
         private const int RequiredMinimumWordsCount = 1;
         private const int MaxWordsCount = 150;
+        private const string TooManyWords = "Internal comments must be 150 words or less";
+        private const string CommentRequired = "Enter internal comments";
+
         private const int ClarificationResponseMaxWordsCount = 300;
         private const string ClarificationResponseRequired = "Enter clarification response";
         private const string ClarificationResponseTooManyWords = "Clarification response must be 300 words or less";
 
-        private const string TooManyWords = "Internal comments must be 150 words or less";
-        private const string CommentRequired = "Enter internal comments";
-
+        private const long MaxFileSize = 5 * 1048576;
```

Comment:
magic number better converted so some static variable in a class?

Reply:
yes and no.. I can rename to something a little more meaniful, otherwise it's just like the other magic strings in that class which are self contained

Reply:
I changed it to a more meaningful name